[2023-04-13T11:44:53.265+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data manual__2023-04-13T11:44:47.421835+00:00 [queued]>
[2023-04-13T11:44:53.276+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data manual__2023-04-13T11:44:47.421835+00:00 [queued]>
[2023-04-13T11:44:53.278+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T11:44:53.281+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-13T11:44:53.283+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T11:44:53.297+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data> on 2023-04-13 11:44:47.421835+00:00
[2023-04-13T11:44:53.303+0000] {standard_task_runner.py:55} INFO - Started process 82 to run task
[2023-04-13T11:44:53.307+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data', 'manual__2023-04-13T11:44:47.421835+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp7_c_2syo']
[2023-04-13T11:44:53.310+0000] {standard_task_runner.py:83} INFO - Job 88: Subtask inject_data
[2023-04-13T11:44:53.383+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data manual__2023-04-13T11:44:47.421835+00:00 [running]> on host ffea11fb791c
[2023-04-13T11:44:53.478+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-13T11:44:47.421835+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-13T11:44:47.421835+00:00
[2023-04-13T11:44:53.481+0000] {sql.py:254} INFO - Executing: 
        INSERT INTO measures (timestamp, temp) VALUES
        (1681386234,
        287.4);
        
[2023-04-13T11:44:53.492+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T11:44:53.679+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T11:44:53.686+0000] {sql.py:375} INFO - Running statement: 
        INSERT INTO measures (timestamp, temp) VALUES
        (1681386234,
        287.4);
        , parameters: None
[2023-04-13T11:44:53.704+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-13T11:44:53.730+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data, execution_date=20230413T114447, start_date=20230413T114453, end_date=20230413T114453
[2023-04-13T11:44:53.769+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-13T11:44:53.789+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
